spring-cloud-kubernetes-commons/src/main/java/org/springframework/cloud/kubernetes/commons/profile/AbstractKubernetesProfileEnvironmentPostProcessor.java

```diff
@@ -60,72 +58,64 @@
 	@Override
 	public void postProcessEnvironment(ConfigurableEnvironment environment, SpringApplication application) {
 
-		final boolean kubernetesEnabled = environment.getProperty("spring.cloud.kubernetes.enabled", Boolean.class,
-				true);
+		application.addInitializers(ctx -> LOG.replayTo(AbstractKubernetesProfileEnvironmentPostProcessor.class));
+
+		boolean kubernetesEnabled = environment.getProperty("spring.cloud.kubernetes.enabled", Boolean.class, true);
 		if (!kubernetesEnabled) {
 			return;
 		}
 		addNamespaceFromServiceAccountFile(environment);
-		if (isInsideKubernetes(environment)) {
```

Comment:
this block has moved to a method - it basically does the same thing, but I moved it. I think it is easier to read the code like this.